torchbench.py

```diff
@@ -1068,13 +1126,15 @@ def run_one_model(
     experiment,
     cos_similarity=False,
     skip_accuracy_check=False,
+    quant_exp=False,
 ):
     tolerance = 1e-4
     # Increase the tolerance for torch allclose
     if is_training and current_device == "cuda" and name in REQUIRE_HIGHER_TOLERANCE:
         tolerance = 1e-3
 
     with pick_grad(name, is_training):
+        original_model = copy.deepcopy(model)
```

Comment:
Hi @jansel, I'm trying to get a speedup of quantized model to floating point model, do you think this is the correct way to do it?

here I did a deepcopy, and in L1158 to L1164 I used the quant_context to quantize the model, then I feed  the two models to `experiment` function

Reply:
See above.  TorchDynamo's cache is on the PyCode object level.  So when you enable `torchdynamo.run()` you might get code objects from the A side of the experiment leaking into the B side.

If you want to compare dynamo to dynamo, I would suggest:
1) measure the first one 30 times (using torchdynamo.optimize())
2) call torchdynamo.reset(), which clears all caches
3) measure the second one 30 times (using torchdynamo.optimize())

The interleaving thing won't work right if multiple sides of the experiment use dynamo.